features/product/viewing_products/viewing_product_enabled_variants_options_only.feature

```diff
@@ -0,0 +1,18 @@
+@viewing_products
+Feature: Viewing product's enabled variants only
+    In order to buy only available products
+    As a Customer
+    I want to see only enabled product variants options
+
+    Background:
+        Given the store operates on a channel named "Web-US" in "USD" currency
+        And the store has a "Super Cool T-Shirt" configurable product
+        And this product has option "Size" with values "Small", "Medium" and "Large"
+        And this product has option "Color" with values "Blue", "Green" and "Yellow"
+        And this product has all possible variants
+        But all the product variants with the "Yellow" color are disabled
+
+    @ui
+    Scenario: Seeing only enabled variants options
+        When I view product "Super Cool T-Shirt"
+        Then I should not be able to select the "Yellow" color option value
```

Comment:
To do what you asked in https://github.com/Sylius/Sylius/pull/11241/files#r405581869 I'd change as follows:

```suggestion
@viewing_products
Feature: Viewing product's enabled variants only
    In order to buy only available products
    As a Customer
    I want to see only enabled product variants options

    Background:
        Given the store operates on a channel named "Web-US" in "USD" currency
        And the store has a "Super Cool T-Shirt" configurable product
        And this product has option "Size" with values "Small", "Medium" and "Large"
        And this product has option "Color" with values "Blue", "Green" and "Yellow"
        And this product has all possible variants
        But all the product variants with the "Yellow" color are disabled
        And the store has a "Extra Cool T-Shirt" configurable product
        And this product has option "Size" with values "Small" and "Large"
        And this product has option "Color" with values "Yellow" and "Blue"
        And this product has all possible variants
        But the "Small" / "Blue" variant of this product is disabled
        And the "Large" / "Yellow" variant of this product is disabled

    @ui
    Scenario: Seeing only enabled variants options
        When I view product "Super Cool T-Shirt"
        Then I should not be able to select the "Yellow" color option value

    @ui
    Scenario: Seeing both values for both options when diagonal variants are available
        When I view product "Extra Cool T-Shirt"
        Then I should be able to select the "Yellow" and "Blue" color option values
        And I should be able to select the "Small" and "Large" size option values
```

What do you think? Is it ok for you?

Reply:
I'm fine with that 👍 But we should keep the scenario with `unavailable` price as well I suppose.

Reply:
I don't understand. Do you mean the `Viewing an unavailable message if selected option combination is unavailable` scenario in the `viewing_different_price_for_different_product_variants_selected_with_options.feature` file? If so I cannot keep it because it's not possibile to select unavailable variants anymore (as I already said here https://github.com/Sylius/Sylius/pull/11241#discussion_r405284886).

Reply:
Ahh wait maybe I understand now. Do you want another scenario to verify that if I choose Small / Blue I get an "unavailable" message?

Reply:
yes, that's correct

Reply:
Done: https://github.com/Sylius/Sylius/pull/11241/commits/df192a303e3f674b113df8261cbb787633b8f637